docs/current/guides/snippets/azure-pipelines-container-instances/main.py

```diff
@@ -0,0 +1,97 @@
+import sys
+import os
+
+import anyio
+import dagger
+from azure.identity import DefaultAzureCredential
+from azure.mgmt.containerinstance import ContainerInstanceManagementClient
+
+# configure container group, name and location
+container_name = "my-app";
+container_group_name = "my-app";
+container_group_location = "eastus";
+resource_group_name = "my-group";
+
+async def main():
+
+    # check for required variables in host environment
+    for var in ["DOCKERHUB_USERNAME", "DOCKERHUB_PASSWORD", "AZURE_SUBSCRIPTION_ID", "AZURE_TENANT_ID", "AZURE_CLIENT_ID", "AZURE_CLIENT_SECRET"]:
```

Comment:
I don't like to repeat referencing variables from strings due to possible typos that are hard to detect. It's better to put them in constants and reference the constants instead.

I'll share a pattern where this can be simplified with an enum (to not require installing anything).

Reply:
Here it is: https://github.com/helderco/dagger/commit/6cb1eb5abd01998fedffca21859964fccb400619